pipeline_dp/pipeline_backend.py

```diff
@@ -132,6 +132,20 @@ def flatten(self, col1, col2, stage_name: str):
         """
         pass
 
+    def annotate(self, col, stage_name: str, **kwargs):
+        """Annotates collection with registered annotators.
+
+        Args:
+          col: input collection which contains tuples (key, accumulator).
+          stage_name: name of the stage.
+          kwargs: additional key-value pair arguments supported by
+          DPEngine.aggregate()
+
+        Returns:
+          The input collection after applying all the annotations in _annotators.
```

Comment:
Nit: ... after applying annotations from all registered annotators.

It's better not to mention `_annotators` which is a private variable of this module (i.e. it's an implementaiton detail), while this function is a public API of this module.